osu.Game.Tournament/Screens/Setup/SetupScreen.cs

```diff
@@ -142,7 +142,7 @@ private void reload()
                     Label = "Background video settings",
                     ButtonText = "Change videos",
                     Description = "Set paths and behaviour of background video display.",
```

Comment:
Everything works fine, but we have to document our application well. These videos and images will be applied after the tournament program is fully restarted.

```suggestion
                    Description = "Set paths and behaviour of background video display. Save and restart to make changes take effect.",
```

Reply:
Restarting to make it work isn't our initially expected behaviour, and I consider it as a problem to be fixed.
The latest few commits add an `Invalidate()` action to reload the background immediately after changes were made to background mappings. This should work better.

Reply:
Understood, ping me if the code is in its final state.

Reply:
Done with changes desired in this PR. It is good to go now 🤓👆